### DISCUSSION TITLE
Supabase's sponsoring policies?

### DISCUSSION THREAD

**Discussion by dorjeozer** [supabase/supabase#26293] [Questions]
Hello,

We're arranging an [Apple Developers' Conference](https://arcticonference.com) on March 2025 and I'm wondering that is Supabase's policy in sponsoring events like this? Never seen any visibility in any of the events I've attended, I'm quite confident that a lot of Swift developers would love Supabase if they just knew about it (for me it was love at first sight)!

Best wishes,
Jesse

**Comment by silentworks** [comment]
It's best that you send these sort of requests/questions to Supabase directly via their support system https://supabase.com/dashboard/support/new
